command/sidechain/registration/register_validator.go

```diff
@@ -148,13 +156,20 @@ func runCommand(cmd *cobra.Command, _ []string) error {
 			result.validatorAddress = event["validator"].(ethgo.Address).String() //nolint:forcetypeassert
 			result.stakeResult = "No stake parameters have been submitted"
 			result.amount = 0
-			foundLog = true
+			foundNewValidatorLog = true
+		}
+
+		if commissionUpdatedABI.Match(log) {
+			event, err := commissionUpdatedABI.ParseLog(log)
+			if err != nil {
+				return err
+			}
 
-			break
+			result.commission = event["newCommission"].(*big.Int).Uint64() //nolint:forcetypeassert
 		}
 	}
 
-	if !foundLog {
+	if !foundNewValidatorLog {
 		return fmt.Errorf("could not find an appropriate log in receipt that registration happened")
```

Comment:
A similar error message must be returned in case commission updated log is not found